src/table/table.jsx

```diff
@@ -27,6 +27,9 @@ const Table = React.createClass({
     selectable: React.PropTypes.bool,
     style: React.PropTypes.object,
     wrapperStyle: React.PropTypes.object,
+    headerTableInlineStyle: React.PropTypes.object,
```

Comment:
> I kinda wanted the naming to be consistent with css classes

I think that the css classes shouldn't be here, let's forget them.
Your example looks good to me.